content/rs/databases/active-active/aa-planning.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: Considerations for planning Active-Active databases
+linktitle: Planning considerations
+description: Information about Active-Active database to take into consideration while planning a deployment, such as compatibility, limitations, and special configuration.
+weight: 22
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+/rs/databases/active-active/aa-planning.md,
+/rs/databases/active-active/aa-planning/,
+]
+
+---
+
+In Redis Enterprise, Active-Active geo-distribution is based on [CRDT technology](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) (conflict-free replicated data type). Compared to databases without geo-distribution, Active-Active databases have more complex replication and networking, as well as a different data type.
+
+Because of the complexities of Active-Active databases, there are  special considerations to keep in mind while planning your Active-Active database.
```

Comment:
```suggestion
Because of the complexities of Active-Active databases, there are special considerations to keep in mind while planning your Active-Active database.
```